db/migrate/20160106233632_convert_classification_name_to_column.rb

```diff
@@ -0,0 +1,20 @@
+class ConvertClassificationNameToColumn < ActiveRecord::Migration
+  class Classification < ActiveRecord::Base
+    belongs_to :tag
+  end
+
+  class Tag < ActiveRecord::Base
+  end
+
+  def up
+    add_column :classifications, :name, :string
+    Classification.all.each do |classification|
+      next unless classification.tag
+      classification.update_attributes(:name => File.split(classification.tag.name).last)
```

Comment:
I guess `update_column` could be an bit faster here.

Also, `File.basename(...)` is equivalent to `File.split(...).last`, if I recall correctly.